src/core/server/capabilities/capabilities_service.ts

```diff
@@ -162,7 +177,8 @@ export class CapabilitiesService {
 
   public start(): CapabilitiesStart {
     return {
-      resolveCapabilities: (request) => this.resolveCapabilities(request, [], false),
+      resolveCapabilities: (request, options) =>
+        this.resolveCapabilities(request, [], !!options?.useDefaultCapabilities),
```

Comment:
NIT: imho `options?.useDefaultCapabilities ?? false` is slightly more explicit.

Reply:
Yep, agree. I'll change, thanks!